documentation/modules/ROOT/pages/basics.adoc

```diff
@@ -41,11 +67,6 @@ curl localhost:8080/hello
 hello
 ----
 
-== Dev UI
```

Comment:
I think this section makes more sense at the end of this page, so I moved it below